satis-explicit-packages.json

```diff
@@ -1,17 +1,11 @@
 {
   "name":      "ingenerator/forks_and_extras",
-  "homepage" : "http://php-packages.ingenerator.com",
+  "homepage" : "https://php-packages.ingenerator.com",
   "repositories": [
-    { "packagist": false},
-    {"type": "vcs", "url":  "https://github.com/acoulton/coding-standards"},
+    {"packagist": false},
     {"type": "vcs", "url":  "https://github.com/craig410/rpage"},
     {"type": "vcs", "url":  "https://github.com/acoulton/kodoctrine"},
-    {"type": "vcs", "url":  "https://github.com/acoulton/doctrine1"},
```

Comment:
Think this is still used by schools & finance? But named doctrine/doctrine1, so it's not obv from the repo name :(

This is the only one I spotted tho.

Reply:
Ah yeah I see it now